testsuite-jpms/src/main/java/io/netty/testsuite_jpms/main/HttpHelloWorldServer.java

```diff
@@ -69,9 +84,12 @@ private HttpHelloWorldServer() {
 
     public static void main(String[] args) throws Exception {
 
+        Security.addProvider(new BouncyCastleProvider());
```

Comment:
Why do we need BouncyCastle?

Reply:
That's really weird. What java version is this?

Reply:
Seeing it as well with Java 11, but it worked with Java 24.

Reply:
Ok, figured it out.

Instead of adding BouncyCastle here, you can add a dependency on the `jdk.crypto.ec` module to `pkitesting`:

```diff
Index: pkitesting/src/main/java/module-info.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/pkitesting/src/main/java/module-info.yml b/pkitesting/src/main/java/module-info.yml
--- a/pkitesting/src/main/java/module-info.yml	(revision 317a5e418c1529f168b7c613747e97ed153a0885)
+++ b/pkitesting/src/main/java/module-info.yml	(date 1749585869254)
@@ -17,3 +17,4 @@
   - module: io.netty.common
   - module: io.netty.buffer
   - module: org.bouncycastle.provider
+  - module: jdk.crypto.ec
```